scalardl-node-client-sdk.js

```diff
@@ -37,34 +37,51 @@ function _createGrpcServices(properties) {
     `${clientProperties.getServerPrivilegedPort()}`;
   const ca = clientProperties.getTlsCaRootCertPem();
   const tlsEnabled = clientProperties.getTlsEnabled();
-  let ledgerClient;
-  let ledgerPrivilegedClient;
+
+  let grpcChannelCredentials;
   if (tlsEnabled) {
-    ledgerClient = new LedgerClient(
-        ledgerClientUrl,
-        grpc.credentials.createSsl(Buffer.from(ca, 'utf8')),
-    );
-    ledgerPrivilegedClient = new LedgerPrivilegedClient(
-        ledgerClientUrl,
-        grpc.credentials.createSsl(Buffer.from(ca, 'utf8')),
-    );
+    if (ca) {
+      // Use custom root CA
+      grpcChannelCredentials =
+        grpc.credentials.createSsl(Buffer.from(ca, 'utf8'));
+    } else {
+      // When no custom root CA is provided to init the SSL/TLS connection,
+      // default root CA maintained by Node.js will be used
+      grpcChannelCredentials =
+        grpc.credentials.createSsl();
```

Comment:
A single line is better if we can keep it less than 80 chars (and there is no policy for doing this).